### DISCUSSION TITLE
Ton Coin Parameters settings

### DISCUSSION THREAD

**Discussion by donPuerto** [Lolliedieb/lolMiner-releases#957] [General]
**If we want a sweetpost of efficiency, we can run lolMiner with the parameter --mclk 810 --cclk 1740, that will reduce Memory Clock to 810 and also lock core with parameter --cclk 1740. Watts will be reduced to 1025 Watts (ca 300W less) with 27.6Ghs.**

Im having trouble where do i put this in bat file?
Any example pls 
thanks

**Comment by qt7621** [comment]
hi,

set "EXTRAPARAMETERS="


set "EXTRAPARAMETERS=--mclk 810 --cclk 1740"
